app.js

```diff
@@ -6,25 +6,32 @@ const path = require('path')
 require('lodash.product')
 const _ = require('lodash')
 const {StatusCodes} = require('http-status-codes')
-const log = console
 const db = require('./db.js')
 
+const log = console
+
 const env = {
   isProd: process.env.NODE_ENV === 'production',
   isTest: process.env.NODE_ENV === 'test'
 }
 
 const config = {
+  verbose: (process.env.VERBOSE === 'true') || true,
```

Comment:
For all of these values with `|| default_value`, all of these are now provided in `.env.sample` (and should be copied across into any existing `.env` files) - therefore, the best way forward is to also remove these defaults from the code itself and mandate the use of environment variables to set these values.

They have been left here in the code as fallback defaults so that this is not a breaking change.

Reply:
@pathikrit are you happy with the above suggestion to remove the defaults from the code, since they are now in the `.env` file?

Reply:
Updated.